package.json

```diff
@@ -51,7 +51,7 @@
   },
   "license": "MIT",
   "scripts": {
-    "test": "babel-node dredd",
+    "test": "babel-node ./node_modules/.bin/dredd",
```

Comment:
The tests failed without this change. dredd isn't looked up from $PATH, it's looked up just in the current directory.